test/e2e/service.go

```diff
@@ -47,129 +46,6 @@ var _ = Describe("Services", func() {
 		namespace1 = "e2e-ns-" + dateStamp() + "-1"
 	})
 
-	It("should provide DNS for the cluster", func() {
```

Comment:
@thockin 
I must say I'm a bit lost. Is DNS the multiport service that is tested here? (tested by dig)

Shall I test a custom service (like a web server created in a pod, with multiple ports)?